src/main/java/co/cask/spanner/sink/SpannerOutputFormat.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.spanner.sink;
+
+import co.cask.cdap.api.data.format.StructuredRecord;
+import co.cask.cdap.api.data.schema.Schema;
+import co.cask.spanner.SpannerConstants;
+import co.cask.spanner.common.SpannerUtil;
+import com.google.cloud.ByteArray;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.Spanner;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.io.NullWritable;
+import org.apache.hadoop.mapreduce.JobContext;
+import org.apache.hadoop.mapreduce.OutputCommitter;
+import org.apache.hadoop.mapreduce.OutputFormat;
+import org.apache.hadoop.mapreduce.RecordWriter;
+import org.apache.hadoop.mapreduce.TaskAttemptContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Spanner output format
+ */
+public class SpannerOutputFormat extends OutputFormat<NullWritable, StructuredRecord> {
+  private static final Logger LOG = LoggerFactory.getLogger(SpannerOutputFormat.class);
+
+  @Override
+  public RecordWriter<NullWritable, StructuredRecord>
+  getRecordWriter(TaskAttemptContext context) throws IOException, InterruptedException {
```

Comment:
I think it's more readable if getRecordWriter(TaskAttemptContext context) is on the previous line and throws is on this line (unless it doesn't fit?)